### DISCUSSION TITLE
Updating ffmpeg

### DISCUSSION THREAD

**Discussion by cooltings** [blakeblackshear/frigate#14179] [Ask A Question]
I recently updated go2rtc and it does not work with the current version of ffmpeg (4. something). I am on a raspberry pi 4 and I run frigate as an addon within home assistant OS. How can I do this?
Thanks.

**Comment by NickM-27** [comment]
The rpi requires a specific ffmpeg build. It works fine with the latest version of go2rtc as long as you're not overwriting the necessary config
